frontend/src/AppRoutes.tsx

```diff
@@ -11,4 +14,12 @@ export const AppRoutes = createBrowserRouter([
     path: "/sign-up",
     element: <SignUpPage />,
   },
+  {
+    path: "/dashboard",
+    element: <Dashboard/>
+  },
+  {
+    path: "/main",
+    element: <Main/>
```

Comment:
At first I created a dashboard page. Later on I was thinking that we will need a to have a reserved spot, such as 'Team', so that the teacher is able to create team. So I made dashboard a component of main, but I did not remove dashboard route.
Thank you for the comments.